src/components/funFacts/FunFactPage.tsx

```diff
@@ -1,10 +1,13 @@
 import React from "react";
+import { funFacts } from "./funfacts"
+import { FunFactComponent } from "./FunFactComponent"
+import "./FunFactPage.scss"
```

Comment:
Semicolons after imports
```suggestion
import { funFacts } from "./funfacts";
import { FunFactComponent } from "./FunFactComponent";
import "./FunFactPage.scss";
```

Reply:
done, thanks